actions/middleware.js

```diff
@@ -0,0 +1,31 @@
+const GET_MIDDLEWARE = 'GET_MIDDLEWARE'
+const ADD_MIDDLEWARE = 'ADD_MIDDLEWARE'
+const RESET_MIDDLEWARE = 'RESET_MIDDLEWARE'
+
+// action creators
+function getMiddleware () {
+  return {type: GET_MIDDLEWARE}
+}
+
+function addMiddleware (middleware) {
+  return {
+    type: ADD_MIDDLEWARE,
+    middleware
+  }
+}
+
+function resetMiddleware () {
+  return {
+    type: RESET_MIDDLEWARE
+  }
+}
+
+// action types
+module.exports = {
+  getMiddleware,
```

Comment:
No need for `getMiddleware` since this is equivalent to `store.getState().middleware`

Reply:
Done